test/Project.toml

```diff
@@ -6,11 +6,18 @@ LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 MPI = "da04e1cc-30fd-572f-bb4f-1f8673147195"
 OrdinaryDiffEq = "1dea7af3-3e70-54e6-95c3-0bf5283fa5ed"
 Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
```

Comment:
```suggestion
```
We removed `Random` a few days ago from the current tests - do you use it somewhere?

Reply:
No, I thought it was there previously. Removing..